setup.py

```diff
@@ -97,7 +97,7 @@
     "isort @ git+git://github.com/timothycrosley/isort.git@e63ae06ec7d70b06df9e528357650281a3d3ec22#egg=isort",
     "flake8",
 ]
-extras["dev"] = extras["testing"] + extras["quality"] + ["mecab-python3<1", "scikit-learn", "tensorflow", "torch"]
+extras["dev"] = extras["testing"] + extras["quality"] + ["mecab-python3<1", "scikit-learn", "tensorflow", "torch>=1.5.0", "keras2onnx==1.7.0", "onnxruntime"]
```

Comment:
Pushing for such kind of dependencies changes will be very hard for me, especially the torch one. We do not have such requirement anywhere else in the lib ...

We'll not be able to merge this PR in the current state, please remove such constraints on the dependencies _(i.e. as they were before)_.